requirements-dev.txt

```diff
@@ -4,25 +4,55 @@
 #
 #    pip-compile requirements-dev.in
 #
-appdirs==1.4.4            # via fs
-attrs==20.3.0             # via pytest
-coverage==5.3             # via -r requirements-dev.in
-defcon==0.7.2             # via -r requirements-dev.in
-fonttools[ufo,unicode]==4.17.1  # via defcon
-fs==2.4.11                # via fonttools
-iniconfig==1.1.1          # via pytest
-lxml==4.6.2               # via xmldiff
-packaging==20.7           # via pytest
-pluggy==0.13.1            # via pytest
-py==1.9.0                 # via pytest
-pyparsing==2.4.7          # via packaging
-pytest-randomly==3.5.0    # via -r requirements-dev.in
-pytest==6.1.2             # via -r requirements-dev.in, pytest-randomly
-pytz==2020.4              # via fs
-six==1.15.0               # via fs, xmldiff
-toml==0.10.2              # via pytest
-ufonormalizer==0.4.2      # via -r requirements-dev.in
-xmldiff==2.4              # via -r requirements-dev.in
+
+appdirs==1.4.4
```

Comment:
Is it considered best practice for Python projects to list transitive dependencies?

Reply:
That's done automatically by pip-compile of the pip-tools package.